### PR TITLE
Start drafting docs to go on the docs site

### PR DESCRIPTION
**Pull request by shelbyspees**
Ready to merge, we should definitely expand these though.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 [![CircleCI](https://circleci.com/gh/honeycombio/honeycomb-opentelemetry-java.svg?style=shield&circle-token=e2f4c30919ecbdbfb095415a6f4114a03dc491a0)](https://circleci.com/gh/honeycombio/honeycomb-opentelemetry-java)
 
-**STATUS: this library is BETA.** You're welcome to try it, and let us know your feedback in the issues!
+**STATUS: this library is BETA.**
+You're welcome to try it, and let us know your feedback in the issues!
```
Comment (shelbyspees) [inline README.md:6]: Started adding [semantic line breaks](https://sembr.org) in a few places.

File: docs/TROUBLESHOOTING.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+it is likely because the auto-instrumentation agent is instrumenting
+your manual SDK.
+Don't use the manual SDK tracer.
+
+## Missing custom spans
+
+If you are not seeing custom spans, added from in-code instrumentation,
+it is likely because you are using incompatible versions of the agent and SDK.
+
+## SDK Usage
```
Comment (vreynolds) [inline docs/TROUBLESHOOTING.md:15]: This is currently under Troubleshooting?

Comment (shelbyspees) [reply docs/TROUBLESHOOTING.md:15]: that was where folks suggested should go as of a few weeks ago. I can move it into its own page instead.

Comment (shelbyspees) [reply docs/TROUBLESHOOTING.md:15]: I'm gonna reorganize these pages a bit because it kinda bugs me now

**Review by vreynolds — APPROVED** [review]

**Pull request merged**